package.json

```diff
@@ -94,7 +93,7 @@
     "proxy-middleware": "^0.15.0",
     "react-datepicker": "^1.4.0",
```

Comment:
shouldn't things like this be dependencies (or peed deps?)

Reply:
what do you mean? they are under dependencies